src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_COUNT = 3;
```

Comment:
do we have only 3 figures?
I think we never get `Rectangle` or `IsoscelesTrapezoid` from `getRandomFigure`